netsnmp.py

```diff
@@ -515,14 +515,22 @@ def create_users(self, users):
                     log.debug("create_users: could not create user: %s: (%s: %s)" % (user, e.__class__.__name__, e))
 
     def sendTrap(self, trapoid, varbinds=None):
-        trapType = SNMP_MSG_TRAP if '-v1' in self.cmdLineArgs else SNMP_MSG_TRAP2
-        pdu = lib.snmp_pdu_create(trapType)
+        if '-v1' in self.cmdLineArgs:
+            pdu = lib.snmp_pdu_create(SNMP_MSG_TRAP)
+            if hasattr(self, 'agent_addr'):
+                pdu.contents.agent_addr = (c_ubyte*4)(*(bytearray([int(x) for x in self.agent_addr.split('.')])))
```

Comment:
Please comment this for future developers.  "Eric is super-awesome at python" does not count.